document/chapters/03-sdf_preconditioner.tex

```diff
@@ -164,8 +163,8 @@ \section{Preconditioning Theory}\label{sec:preconditioner_theory}
              }
              
 Using these methods, signed distance fields can act as a preconditioning tool
-for the relatively expensive ray fire process to accelerated CAD-based MCRT by
-using an $O(1)$ process to establish that the conditions of a geometric query
+for the relatively expensive ray fire process to accelerate CAD-based MCRT by
+using an $O(1)$ process by establishing that the conditions of a geometric query
```

Comment:
this uses two phrases starting with "by" in a row - is that what you want?

Reply:
Definitely not. That's clunky. Changing that back.

Reply:
Also do you think using "CAD-based MCRT" at this point in the document is ok given that I've been more explicit about it previously or should I try to be consistent throughout? 

(I'm fine with either but CAD-based MCRT on tessellated models is rather wordy to use frequently.)

Reply:
Maybe establish a new acronym for this early one?  Importantly, your work may be interesting/relevant on any tesselated model - CAD-based or not...

"MRCT-t" for "MCRT on tesselated models"?

Reply:
I like it. And good point about separation of the CAD-based qualifier. I think of those as one in the same too often probably.